demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -129,6 +130,26 @@ def _get_pack_file_path(self, file_name=''):
         """Returns the full file path to pack's file"""
         return os.path.join(self.pack_path, file_name)
 
+    def _get_pack_latest_rn_version(self):
+        """
+        Extract all the Release notes from the pack and reutrn the highest version of release note in the Pack.
+
+        Return:
+            (str): The lastest version of RN.
+        """
+
+        try:
+            list_of_files = glob.glob(self.pack_path + '/ReleaseNotes/*')
+            list_of_release_notes = [os.path.basename(file) for file in list_of_files]
+            list_of_versions = [rn[:rn.rindex('.')].replace('_', '.') for rn in list_of_release_notes]
+        except Exception:
```

Comment:
Do not catch bare Exception. Catch the exception that you need to catch.
log it to debug log/exception log.

Do not return empty string as it both successful and failed flows.